### DISCUSSION TITLE
ETA on release of psycopg 3.1?

### DISCUSSION THREAD

**Discussion by ProfDoof** [psycopg/psycopg#358] [Q&A] [answered]
I was poking around, looking at using this library and noticed that you guys had support for Enums by using the EnumInfo. However, that won't be available until the release of 3.1 so I was checking in on the timeline on that. Thanks for all the work you guys have done on this, it seems like a really useful project.

**Comment by dvarrazzo** [comment, marked as answer]
Hey, sorry, release should have been about a month ago, but stuff happened (such as #350) and has delayed it. I've just finished penning the release notes, so it should be in the next few days.
